websites/R/Radio Garden/presence.ts

```diff
@@ -2,42 +2,45 @@ const presence = new Presence({
 	clientId: "687070418804408445" //The client ID of the Application created at https://discordapp.com/developers/applications
 });
 
-let details: HTMLInputElement, state: HTMLInputElement;
+let details: HTMLInputElement, state: HTMLInputElement, currentState: string;
 
 presence.on("UpdateData", async () => {
 	if (!document.title.includes("Explore live radio by rotating")) {
-		details = document.querySelector(".ChannelTitle_title__2QQj5");
-		state = document.querySelector(".ChannelTitle_subtitle__DZ_ZQ");
+		details = document.querySelector("._title_165lg_36"); // which radio we're tuning right now
+		state = document.querySelector("._subtitle_165lg_43"); // the state where the radio serves
```

Comment:
```suggestion
		details = document.querySelector("._title_165lg_36")?.textContent ?? "Unknown Radio"; // which radio we're tuning right now
		state = document.querySelector("._subtitle_165lg_43")?.textContent ?? "Unknown Region/Country"; // the state where the radio serves
```

Reply:
How about this? From what I could see, the only way the element isn't there is when the site is currently loading.
<details>
<summary>Screenshot of the loading page</summary>
<br>

![Screenshot 2022-02-07 060441](https://user-images.githubusercontent.com/45804108/152705308-210aadbf-a47a-495c-9fac-8dba9ea365e0.jpg)

</details>

```suggestion
		details = document.querySelector("._title_165lg_36")?.textContent ?? "Loading..."; // which radio we're tuning right now
		state = document.querySelector("._subtitle_165lg_43")?.textContent ?? "Loading..."; // the state where the radio serves
```

Reply:
I think you can understand it better that way:
```suggestion
		details = document.querySelector("._title_165lg_36")?.textContent ?? "Loading Radio..."; // which radio we're tuning right now
		state = document.querySelector("._subtitle_165lg_43")?.textContent ?? "Loading Region/Country..."; // the state where the radio serves
```

Reply:
I just realized the `if` just below `.on('UpdateData')` actually checks if the site still loading or not. So I reverted back to your first suggestion. Do please take a look at the commit that I just pushed.

Thanks in advance.

Reply:
👍